packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -143,18 +143,19 @@ export const FileSnapshotWriterClassFactory = <TBase extends ReaderConstructor>(
             return super.readBlob(sha);
         }
 
-        public async getVersions(versionId: string, count: number): Promise<api.IVersion[]> {
+        public async getVersions(versionId: string | null, count: number): Promise<api.IVersion[]> {
             // If we already saved document, that means we are getting here because of snapshot generation.
             // Not returning tree ensures that ContainerRuntime.snapshot() would regenerate subtrees for
             // each unchanged data store.
             // If we want to change that, we would need to capture docId on first call and return this.latestWriterTree
             // when latest is requested.
+            if (this.latestWriterTree && (this.docId === versionId || versionId === undefined) || versionId === null) {
+                return [{ id: "latest", treeId: FileStorageVersionTreeId }];
+            }
+
             if (this.docId === undefined && versionId !== undefined) {
```

Comment:
This condition needs to be changed as well to exclude null (you can change undefined -> null given that we no longer support undefined)